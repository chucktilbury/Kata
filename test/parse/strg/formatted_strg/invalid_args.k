;-------------------------------------------------------------------------------
; Test using a formatted string with invalid args in a string expression
;
; Missing a comma between args. Produce a syntax error in the context of a
; formatted string.
;-------------------------------------------------------------------------------

var string some_name = "this is a string"(1 2, 3)
