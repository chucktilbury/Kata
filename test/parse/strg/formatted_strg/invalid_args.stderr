Syntax: /home/chuck/Src/Kata/test/parse/strg/formatted_strg/invalid_args.k: 8: 46: expected a ',' or a ')' but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/strg/formatted_strg/invalid_args.k: 8: 46: expected module item or end of file but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/strg/formatted_strg/invalid_args.k: 8: 46: module cannot be empty
